app/views/user_mailer/reset_password_instructions.text.erb

```diff
@@ -0,0 +1,8 @@
+Hello,
+
+Someone (hopefully you) has requested a link to change your password. You can do this through the link below.
+
+<%= edit_user_password_url(reset_password_token: @token) %>
+
+If you didn't request this, please ignore this email.
+Your password won't change until you access the link above and create a new one.
```

Comment:
Is this the default content from Devise? And I can't find any HAML/ERB references in the ADRs – is it just an ambient property of this repo that we:

- Use HAML for HTML and XML
- Use ERB for mailers?

If so should we make that explicit in an ADR separate from this? I'm sure there's Reasoning for it.

Reply:
More or less - I wanted to add the "hopefully you" parenthetical, because I like it and it sounds marginally less alarming than the default "Someone".

If there is reasoning, it might be lost to time. I don't know if we want to add a retroactive ADR, but whether or not we do doesn't seem to be tied to this feature branch (since the mixture is already there).

Reply:
I *thought* the wording was better than the default! (I also like your add). Do we think it likely that they'll want to edit this content themselves and hence might want to provision a Notify template?

Reply:
Let's ask them!

Reply:
Ok. Let's not let it block a merge; if it's important, it'll come back.